### ISSUE TITLE
Get being banned

### ISSUE THREAD

**Issue by Samigos** [iam4x/pokemongo-webspoof#387]
The last 3-4 days, the minute I use the hack, I get banned! (can't catch pokemon or use pokestops)
Why is that happening?

**Comment by MoOx** [comment]
Did you try in another country? You should be careful about the location you use.

**Comment by Samigos** [comment]
@MoOx No.. I'm moving inside the town I live in and sometimes, in nearby cities...

**Comment by MoOx** [comment]
Sorry, I can't try, since I update to latest iOS 10 beta, I can't build the app via xcode...

**Comment by adrianoresende** [comment]
@Samigos You use another hack? If no, the cause can be stopped without changing the GPS position or no.

@MoOx The application changes small details of position while you are standing? If not, it's not human.

**Comment by asonephady** [comment]
This hasn't happened to me. I was able to use the it fine last few days.

**Comment by iam4x** [comment]
Fine for me aswell 👍

**Comment by adrianoresende** [comment]
@iam4x What do you think about the application changes small details of position while you are stopped?

**Comment by MoOx** [comment]
@adrianoresende it's already implemented. See https://github.com/iam4x/pokemongo-webspoof/pull/358

**Comment by kashx** [comment]
@MoOx you can build the app by downloading xcode beta

**Comment by MoOx** [comment]
I already have Xcode beta.
